PWGDQ/TableProducer/tableMaker.cxx

```diff
@@ -226,8 +228,8 @@ struct TableMaker {
   }
 
   // Templated function instantianed for all of the process functions
-  template <uint32_t TEventFillMap, uint32_t TTrackFillMap, uint32_t TMuonFillMap, typename TEvent, typename TTracks, typename TMuons, typename TAmbiTracks, typename TAmbiMuons>
-  void fullSkimming(TEvent const& collision, aod::BCs const& bcs, TTracks const& tracksBarrel, TMuons const& tracksMuon, TAmbiTracks const& ambiTracksMid, TAmbiMuons const& ambiTracksFwd)
+  template <uint32_t TEventFillMap, uint32_t TTrackFillMap, uint32_t TMuonFillMap, typename TEvent, typename TTracks, typename TMuons, typename TAmbiTracks, typename TAmbiMuons, typename TDalitzBits>
+  void fullSkimming(TEvent const& collision, aod::BCs const& bcs, TTracks const& tracksBarrel, TMuons const& tracksMuon, TAmbiTracks const& ambiTracksMid, TAmbiMuons const& ambiTracksFwd, TDalitzBits const& dalitzBits)
```

Comment:
In the first version of the task the track and dalitzbit table were joinable, but now there are not anymore.
The reason for that is that a lots of Dalitzbits are 0. Now we fill in the table only non-0 dalitzbits, and keep the track index, which reduces the size of the table. Another advantage of that is that in the dalitzSelection task we can apply filters (if we wanted a joinable table we would need to run on all tracks to tag them, including the ones which does not pass the filter).
Moreover, we also keep the event index, so that the event iterator allows to take only the dalitzbits associated to the event.